test_pr_quick_start_guide.rst

```diff
@@ -0,0 +1,91 @@
+*****************************
+Quick-start development guide
```

Comment:
```suggestion
Quick-start development guide
```
I tried to figure out how we could change this title but failed. Any ideas?

Reply:
Test Pull Request locally guide?
Quick-start local PR testing guide?
Neither one looks good to me :(

Reply:
maybe
```
How to test another's PR locally
```
? (to highlight the case)

Reply:
How about `How to test a PR locally` instead? Or write out `another's` as `another person's`?

Reply:
sounds good to me:)

Reply:
both

Reply:
@Andersson007 @felixfontein This and other conversations have been fixed in https://github.com/ansible/community-docs/pull/16/commits/e7d32b730e343d177fa1d2444bf427c0c1747202,
can you please take another look?